sakura/sakura/pubsub/rabbitmq/rabbitmq_client/client.py

```diff
@@ -0,0 +1,193 @@
+import asyncio
+from contextlib import asynccontextmanager
+from logging import getLogger
+from typing import Any, Optional
+
+import aio_pika
+from aio_pika.abc import (
+    AbstractExchange,
+    AbstractIncomingMessage,
+    AbstractQueue,
+    AbstractRobustChannel,
+    AbstractRobustConnection,
+    DeliveryMode,
+)
+from aio_pika.pool import Pool
+
+from sakura.pubsub.rabbitmq.rabbitmq_client.settings import Settings
+from sakura.pubsub.rabbitmq.rabbitmq_client.types import Exchange, Queue
+
+logger = getLogger(__name__)
+
+
+class RabbitMQClient:
+    Settings = Settings
```

Comment:
Find a way to make this apparent this is a class, declare here or maybe declare with inherit and pass

Reply:
I've changed the Settings object name to RabbitMQClientSettings,
it's still not apparent that it's a class but at least it differs from the "Settings" in `client.py`